client/my-sites/plugins/plugin-autoupdate-toggle/index.jsx

```diff
@@ -16,12 +16,17 @@ module.exports = React.createClass( {
 	displayName: 'PluginAutopdateToggle',
 
 	propTypes: {
+		isMock: React.PropTypes.bool,
 		site: React.PropTypes.object.isRequired,
 		plugin: React.PropTypes.object.isRequired,
 		wporg: React.PropTypes.bool
 	},
 
 	toggleAutoupdates: function() {
+		if( this.props.isMock ) {
```

Comment:
I fear this would complicate components by adding these early returns. What if we just do a css display: none for this limited use?